pymc/stats/log_density.py

```diff
@@ -126,46 +128,35 @@ def compute_log_density(
     if kind not in ("likelihood", "prior"):
         raise ValueError("kind must be either 'likelihood' or 'prior'")
 
+    # We need to disable transforms, because the InferenceData only keeps the untransformed values
+    umodel = remove_value_transforms(model)
```

Comment:
This kind of change is why this PR was done. A similar thing was taking place in https://github.com/pymc-devs/pymc-experimental/pull/345#discussion_r1627311093